src/log-store/src/raft_engine/log_store.rs

```diff
@@ -192,11 +192,18 @@ impl LogStore for RaftEngineLogStore {
             return Ok(AppendBatchResponse::default());
         }
 
+        // Records the last entry id for each region's entries.
+        let mut last_entry_ids = HashMap::with_capacity(entries.len());
         let mut batch = LogBatch::with_capacity(entries.len());
 
         for e in entries {
             self.check_entry(&e)?;
+            // For raft-engine log store, the namespace id is the region id.
             let ns_id = e.namespace_id;
+            last_entry_ids
```

Comment:
Maybe add an UT to assure this change.

Reply:
Added a test to ensure the returned last entry id for each region is the expected one.